src/app/register/success/page.tsx

```diff
@@ -27,10 +27,12 @@ export default function RegisterSuccessPage() {
             <CardTitle>Registration Successful!</CardTitle>
           </CardHeader>
           <CardContent>
-            <CardDescription>You can now log in and start using Czechitoken.</CardDescription>{" "}
+            <CardDescription>You can now log in and start using Czechitoken.</CardDescription>
           </CardContent>
           <CardFooter>
-            <Button className="w-full">Log in</Button>
+            <a href="/" className="w-full">
```

Comment:
Please use `<Link></Link>` from  `import Link from "next/link";` instead of `<a></a>